src/packages/dictionary/dictionary/workspace/dictionary-workspace-editor.element.ts

```diff
@@ -38,10 +38,16 @@ export class UmbDictionaryWorkspaceEditorElement extends UmbLitElement {
 		return html`
 			<umb-workspace-editor alias="Umb.Workspace.Dictionary">
 				<div id="header" slot="header">
-					<uui-button href="/section/dictionary/dashboard" label="Back to list" compact>
+					<uui-button
+						href="/section/dictionary/dashboard"
```

Comment:
This url seems to be absolute. I'm not sure it would work inside the CMS, where we depend on links being relative, so they pick up the base URL. Try it out and consider changing it to `section/dictionary/dashboard`.